.env.example

```diff
@@ -0,0 +1 @@
+PORT=8080
```

Comment:
do you use PORT env variable in your app?

Reply:
not yet, I added it in case we will turn this into an API

Reply:
It's usually better to add things when needed, otherwise they can be misleading if they exist and are not used.